packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts

```diff
@@ -134,8 +142,8 @@ export class ChainAdapter
         addressNList,
         tx: utx,
         chain_id: caip2.ChainReference.CosmosHubMainnet,
-        account_number: account.chainSpecific.accountNumber,
-        sequence: account.chainSpecific.sequence
+        account_number: String(account.chainSpecific.accountNumber),
+        sequence: String(account.chainSpecific.sequence)
```

Comment:
According to typings, these should already come as a string so no need to cast them